metrics/metricstest/metrics_mock.go

```diff
@@ -0,0 +1,129 @@
+package metricstest
+
+import (
+	"github.com/prebid/prebid-cache/config"
+	"github.com/prebid/prebid-cache/metrics"
+	"time"
+)
+
+/*Define Mock metrics        */
+var HT1 map[string]float64
```

Comment:
What is `HT1` vs `HT2`?

Reply:
Hash table 1 and hash table 2. Let me change it to `MockHistograms` and `MockCounters` respectively.